src/components/modules/contacts/server/contact.firebase.ts

```
@@ -0,0 +1,162 @@
+import { ContactPayload } from "@/@types/contact.entity";
+import { db } from "@/core/config/firebase/firebase";
+import {
+  collection,
+  doc,
+  getDoc,
+  getDocs,
+  query,
+  serverTimestamp,
+  updateDoc,
+  where,
+  deleteDoc,
```

ESLint

[error] 'deleteDoc' is defined but never used.

(@typescript-eslint/no-unused-vars)